### ISSUE TITLE
Failing interaction between module declaration and @doc

### ISSUE THREAD

**Issue by sethaxen** [domluna/JuliaFormatter.jl#160]
On a fresh Julia 1.3 install with the latest JuliaFormatter, I have a file `test.jl` containing

```julia
module MyModule

import Markdown: @doc_str

@doc doc"""
    foo()
"""
foo() = bar()

end # module
```

When I format this, I get the following error:
```julia
julia> format_file("test.jl")
ERROR: BoundsError: attempt to access 2-element Array{CSTParser.EXPR,1} at index [3]
Stacktrace:
 [1] getindex at ./array.jl:744 [inlined]
 [2] getindex(::CSTParser.EXPR, ::Int64) at /Users/saxen/.julia/packages/CSTParser/TPWx4/src/spec.jl:419
 [3] p_macrocall(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:657
 [4] pretty(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:387
 [5] #p_block#30(::Bool, ::Bool, ::Bool, ::typeof(JuliaFormatter.p_block), ::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:724
 [6] p_block(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:718
 [7] pretty(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:335
 [8] p_module(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:879
 [9] pretty(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:337
 [10] pretty(::CSTParser.EXPR, ::JuliaFormatter.State) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/pretty.jl:446
 [11] #format_text#64(::Int64, ::Int64, ::Bool, ::Bool, ::Bool, ::typeof(format_text), ::String) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/JuliaFormatter.jl:249
 [12] #format_text at ./none:0 [inlined]
 [13] #format_file#65(::Bool, ::Bool, ::Int64, ::Int64, ::Bool, ::Bool, ::Bool, ::typeof(format_file), ::String) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/JuliaFormatter.jl:335
 [14] format_file(::String) at /Users/saxen/.julia/packages/JuliaFormatter/gMKVZ/src/JuliaFormatter.jl:329
 [15] top-level scope at REPL[26]:1
```

Removing either the module declaration or `@doc doc` causes the error to go away.

Version info:
```julia
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.0.0)
  CPU: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)

(v1.3) pkg> st JuliaFormatter
    Status `~/.julia/environments/v1.3/Project.toml`
  [98e50ef6] JuliaFormatter v0.1.43
```

**Comment by domluna** [comment]
So it looks like this may be a bug with CSTParser. It some cases the @doc macrocall is parsed with a third arg and sometimes it's not.

For have I have a simple fix to get around this in #161

**Comment by domluna** [comment]
If another case doesn't work keep the issues coming :)
